From 2d68635da7455dc9e36161aebba58a7062682d98 Mon Sep 17 00:00:00 2001
From: Ben Mares <15216687+maresb@users.noreply.github.com>
Date: Wed, 21 Apr 2021 02:01:13 +0200
Subject: [PATCH] =?UTF-8?q?setup.py=20fix=20sklearn=20=E2=86=92=20scikit-l?=
 =?UTF-8?q?earn?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The package name is `scikit-learn` while the import is `sklearn`.
See https://pypi.org/project/sklearn/ for the official recommendation.
Requiring `sklearn` can lead to subtle problems as explained in
https://github.com/scikit-learn/scikit-learn/issues/8215 .
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 2cef6fd..8a8b657 100644
--- a/setup.py
+++ b/setup.py
@@ -42,7 +42,7 @@ _install_requires = [
     "numpy",
     "pandas",
     "six",
-    "sklearn",
+    "scikit-learn",
     "matplotlib",
     "graphviz",
     "dm-sonnet",
-- 
2.26.2

